commit bbf662aaddddc5f3f588b3edd47dc7be16b990f2
Author: William wu <william.wu@rock-chips.com>
Date:   Tue Jan 17 15:32:07 2017 +0800

    usb: host: xhci: plat: check hcc_params after add hcd
    
    commit 5de4e1ea9a731cad195ce5152705c21daef3bbba upstream.
    
    The commit 4ac53087d6d4 ("usb: xhci: plat: Create both
    HCDs before adding them") move add hcd to the end of
    probe, this cause hcc_params uninitiated, because xHCI
    driver sets hcc_params in xhci_gen_setup() called from
    usb_add_hcd().
    
    This patch checks the Maximum Primary Stream Array Size
    in the hcc_params register after add primary hcd.
    
    Signed-off-by: William wu <william.wu@rock-chips.com>
    Acked-by: Roger Quadros <rogerq@ti.com>
    Fixes: 4ac53087d6d4 ("usb: xhci: plat: Create both HCDs before adding them")
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1310_bbf662aaddddc5f3f588b3edd47dc7be16b990f2.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1310_bbf662aaddddc5f3f588b3edd47dc7be16b990f2.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
